docs/source/status.rst

```diff
@@ -169,18 +170,18 @@ Notes:
 C Data Interface
 ================
 
-+-----------------------------+-------+--------+-------+
-| Feature                     | C++   | Python | R     |
-|                             |       |        |       |
-+=============================+=======+========+=======+
-| Schema export               | ✓     | ✓      | ✓     |
-+-----------------------------+-------+--------+-------+
-| Array export                | ✓     | ✓      | ✓     |
-+-----------------------------+-------+--------+-------+
-| Schema import               | ✓     | ✓      | ✓     |
-+-----------------------------+-------+--------+-------+
-| Array import                | ✓     | ✓      | ✓     |
-+-----------------------------+-------+--------+-------+
++-----------------------------+-------+--------+-------+-------+
```

Comment:
@jorgecarleitao this one's your baby ❤️, may you please populate what's supported on the C Data interface in Rust.
Please push directly on my branch